alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -2884,4 +2884,17 @@ private static function activateConvertProfileJob($entryId, $localFilePath)
 			return false;
 	}
 
+	public static function handleLiveEntryChanged(LiveStreamEntry $liveEntry)
```

Comment:
Can you explain the why was this addition needed? 
You are not checking if these values were changed but, only that they exist.